### PR TITLE
Fix definition of GQ vs non-GQ immigrant

### PR DESCRIPTION
**Pull request by zmbc**
### Description

- *Category*: bugfix
- *JIRA issue*: none
- *Research reference*: none

### Changes and notes

I had a typo in "institutionalized" -- I have replaced this with using the newly loaded "household_type" column, which is clearer.

I will squash this commit with #169 before we merge the epic into main.

### Verification and Testing

Stepped through and verified that gq_immigrants had the expected relationships, and non_gq_immigrants had the expected relationships.

### CODE REVIEW

File: src/vivarium_census_prl_synth_pop/components/immigration.py [hunk 1]
```diff
@@ -29,10 +29,7 @@ def setup(self, builder: Builder):
         immigrants = persons_data[persons_data["immigrated_in_last_year"]]
 
         is_gq = immigrants["relation_to_household_head"].isin(
-            [
-                "Insitutionalized GQ pop",
-                "Noninstitutionalized GQ pop",
-            ]
+            data_values.GROUP_QUARTER_IDS.keys()
```
Comment (rmudambi) [inline src/vivarium_census_prl_synth_pop/components/immigration.py:32]: I thought this was supposed to key off of the new housing type column?

Comment (zmbc) [reply src/vivarium_census_prl_synth_pop/components/immigration.py:32]: That column is in the households, not persons, data. In other places, we use the relationship column for this purpose when working with the persons data. I suppose to be more technically correct we could always join to the households data and use the newly-loaded column, but that might overcomplicate things.

Just to note: the reason for adding that column in #167 was not to use it for immigration; it was to fix an unrelated bug in our handling of the households data.

Comment (rmudambi) [reply src/vivarium_census_prl_synth_pop/components/immigration.py:32]: To clarify, this column should be correct once we're in the simulation. The issue is that we couldn't rely on this column in the source ACS data-set. Am I understanding this correctly? If so, feel free to merge.

**Review by rmudambi — APPROVED** [review]

File: src/vivarium_census_prl_synth_pop/components/immigration.py (hunk 1 above)
Comment (zmbc) [reply src/vivarium_census_prl_synth_pop/components/immigration.py:32]: Nope not quite, and I see how this is confusing.

I think what you said is a clearer way to think about it, and I will update this code to use the newly-loaded housing type column from ACS.

**Comment by zmbc** [comment]
@rmudambi I've essentially completely replaced this with using the `household_type` column since I agree that is clearer. I also tested this again to ensure it is actually working.

**Review by rmudambi — APPROVED** [review]

File: src/vivarium_census_prl_synth_pop/components/immigration.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    @property
+    def name(self):
+        return "immigration"
+
+    #################
+    # Setup methods #
+    #################
+
+    def setup(self, builder: Builder):
+        persons_data = builder.data.load(data_keys.POPULATION.PERSONS)
```
[the remark this answers is not in the document]
Comment (zmbc) [reply src/vivarium_census_prl_synth_pop/components/immigration.py:29]: I was thinking about this too! I like your solution, and I don't think it's too much trouble.

**Pull request merged**